.travis.yml

```diff
@@ -3,8 +3,13 @@ go:
     - 1.7
     - 1.8
     - tip
-script: go test -v -randtests 1000 ./...
+script:
+    - PKGS=$(go list ./... | grep -v /vendor/)
```

Comment:
Setting the `PKGS` and `GO_FILES` variables should be moved into [`before_script`](https://docs.travis-ci.com/user/customizing-the-build) so as to not pollute the travis logs. I'm quite sure that should still work as now.